projects/plugins/jetpack/extensions/blocks/contact-form/edit.js

```diff
@@ -71,8 +74,17 @@ export function JetpackContactFormEdit( {
 		customThankyouRedirect,
 		jetpackCRM,
 		formTitle,
+		salesforceData,
 	} = attributes;
 
+	const isSalesForceExtensionEnabled = !! window?.Jetpack_Editor_Initial_State?.available_blocks[
+		'contact-form/salesforce-lead-form'
+	];
+
+	if ( isSalesForceExtensionEnabled ) {
+		variations = [ ...variations, salesforceLeadFormVariation ];
+	}
+
```

Comment:
Shouldn't we move this logic to the `variations.js` file?
By doing that, we ensure that wherever the `variations` are used, it'll contain all existent variations.
Right now, we can see the Salesforce variation on the placeholder, but if you search in the Block Library, you won't find it.
That's because the variations we see in the Library are registered by the `index.js` file.
Also, if we move this logic, we could reutilize the `defaultBlockStyling` introduced by you a few days ago.

Reply:
Yes to all of the above, but that would also involve the feature flag to be read on more than one place. Right now I want this hidden for internal testing, and this check makes for a single point of entry for the variation. Once we get this reviewed and tested, we'd remove the flag and make this into the main variations file, alongside the defaultBlockStyling optimization.

I'm not sure it's the best way, but it made sense to me while working in it. Let me know if you think otherwise.

Reply:
If not having the Salesforce template available in the Library isn't an issue now, I'll give you the ✅.
But I'll leave it to @digitalwaveride to decide.